src/containers/item/item-content.svelte

```diff
@@ -0,0 +1,133 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Labeled from 'ui/labeled.svelte';
+  import RadioGroup from 'ui/radio-group.svelte';
+  import ChipGroup from 'ui/chip-group.svelte';
+  import Button from 'ui/button.svelte';
+  import Snackbar from 'ui/snackbar.svelte';
+  import PurchaseModal from '@/components/item/purchase-modal.svelte';
+
+  import {isAuthed, user} from '@/store/user';
+  import {openModal} from '@/store/modal';
+  import {open as snackbarOpen, closeSnackbar} from '@/store/snackbar';
+
+  export let name;
+  export let type;
+  export let description;
+  export let varieties;
+  export let variety;
+  export let inSizes;
+  export let price;
+  export let purchases;
+
+  let dispatch = createEventDispatcher();
+
+  $: sizes = inSizes ? Object.keys(variety.sizes).map(size => ({
+    label: size,
+    disabled: variety.sizes[size] <= 0,
+  })) : null;
+  const changeVariety = e => {
+    chosenSize = null;
+    dispatch('change', e.detail);
+  };
+
+  let chosenSize = null;
+  let err = false;
+  let animation = false;
+  const chooseSize = e => chosenSize = e.detail;
+  const purchase = () => {
+    if (!inSizes || (inSizes && chosenSize)) {
```

Comment:
```suggestion
    if (!inSizes || chosenSize) {
```
looks like someone doesn't remember ITP1, when we had the "implies" operator 😜

Reply:
Omg, changes much of my future code))